vispy/gloo/program.py

```diff
@@ -301,7 +301,10 @@ def __setitem__(self, name, data):
             elif kind == 'attribute':
                 # Is this a constant value per vertex
                 is_constant = False
-                isscalar = lambda x: isinstance(x, (float, int))
+
+                def isscalar(x):
+                    return isinstance(x, (float, int))
+
```

Comment:
Are lambda's always considered bad? I cannot imagine this was the only lambda in our code base ...